src/discover.ts

```diff
@@ -107,35 +98,24 @@ export class TestFileDiscoverer extends vscode.Disposable {
 
     await Promise.all(
       vscode.workspace.workspaceFolders.map(async (workspaceFolder) => {
-        const workspacePath = workspaceFolder.uri.fsPath
-        const exclude = this.config.exclude
-        for (const include of this.config.include) {
-          const pattern = new vscode.RelativePattern(
-            workspaceFolder.uri,
-            include,
+        const files = await globFiles(this.config.include, this.config.exclude, workspaceFolder.uri.fsPath)
```

Comment:
Should use use `this.config.dir || workspaceFolder.uri.fsPath` for better performance?

Reply:
Great idea.

Reply:
You marked this as resolved, but it's not

```suggestion
        const files = await globFiles(this.config.include, this.config.exclude, this.config.dir || workspaceFolder.uri.fsPath)
```

Reply:
Sorry - now I got it! :)